.github/workflows/publish-cdn.yml

```diff
@@ -52,9 +52,19 @@ jobs:
           aws-region: us-east-1
 
       - name: Copy files to CDN
+        shell: bash
         run: |
-          cp dist/constructorio-ui-autocomplete-bundled dist/${{ inputs.version }}.js
-          aws s3 sync dist/${{ inputs.version }}.js s3://constructor-frontend-dev/ui/autocomplete --cache-control 'public, max-age=1800' --acl 'public-read'
+          # remove the "v" prefix from the version
+          current_version=$(echo "${{ inputs.version }}" | tr -d 'v')
```

Comment:
**Suggestion**: @guiquintelas small suggestion here, but maybe we could have an action (e.g. just like the `compile` one) to build this and throw it into the `env`?

I can see that we're doing the same thing in 3 places already, so it might be a bit flaky to maintain

Reply:
I tried implementing with a shared `env`, but we would lose our failsafe when manually running jobs. If anyone accidentally fills in the version with a prefix, we can safely handle it.

So I went ahead and created a new action just for it.

Done 6ce74b6017d7ff9b47ea0256031a4d69cffc8bae ✅